### ISSUE TITLE
API Server quotes "unauthorized"

### ISSUE THREAD

**Issue by gianni2k** [nicolindemann/coinbase-moneymoney#4]
![image](https://user-images.githubusercontent.com/12541224/32837314-5ba73084-ca0d-11e7-9b34-231c477ee18d.png)

![image](https://user-images.githubusercontent.com/12541224/32837346-7bfb97e4-ca0d-11e7-93a0-94d06e0bacaf.png)

**Comment by nicolindemann** [comment]
Please try to redo the coinbase-section of the account setup.

https://github.com/nicolindemann/coinbase-moneymoney#coinbase

**Comment by gianni2k** [comment]
Hi Nico,

I tried that 4 times already. Even with different API Keys.
Do you know what else I could try?

Regards
Giampiero

**Comment by nicolindemann** [comment]
Still having problems?

**Comment by gianni2k** [comment]
No, all fine. 
Solved it by giving more permissions.

Thank you.

**Comment by lenikey** [comment]
Hey - somehow I am having the exact same issues. Played around several times with API settings and permissions but always get Unauthorized.
I am using the version I downloaded here https://moneymoney-app.com/extensions/ and Moneymoney Version 2.3.9 (Beta). 
I also tried to use the version from Github, however it doesn't give me the option to chose Coinbase when trying to create a new account. Pls help <3
